eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs

```diff
@@ -429,7 +429,7 @@ rdptrInv txBody (RdmrPtr Cert idx) =
   Certifying <$> fromIndex idx (txBody ^. certsTxBodyL)
 
 getMapFromValue :: MaryValue crypto -> Map.Map (PolicyID crypto) (Map.Map AssetName Integer)
-getMapFromValue (MaryValue _ m) = m
+getMapFromValue (MaryValue _ (MultiAsset m)) = m
```

Comment:
Actually it doesn't seem to be used at all.. But we still need to keep it around, because someone using us might be calling it?

Reply:
You are right, in fact, it is not used anywhere in IOG: https://github.com/search?q=org%3Ainput-output-hk+getMapFromValue&type=code

Just in case someone outside organization is using it slap `DEPRECATED` pragma on it, so we can phase it out.
```haskell
{-# DEPRECATED getMapFromValue "No longer used" #-}
```